install.sh

```diff
@@ -260,6 +260,34 @@ set_synchrony_url() {
   fi
 }
 
+enable_tcp_protocol() {
+  local install_bitbucket
+  local region
+  local load_balancer_dns
+  local load_balancer_name
+  local original_instance_port
+
+  install_bitbucket=$(get_product "bitbucket" "${CONFIG_ABS_PATH}")
+
+  if [ -n "${install_bitbucket}" ]; then
+    region=$(get_variable 'region' "${CONFIG_ABS_PATH}")
+    load_balancer_dns=$(terraform output | grep '"load_balancer_hostname" =' | sed -nE 's/^.*"(.*)".*$/\1/p')
+    load_balancer_name=$(echo "$load_balancer_dns" | cut -d '-' -f 1)
+    original_instance_port=$(aws elb describe-load-balancers --load-balancer-name "$load_balancer_name" --query 'LoadBalancerDescriptions[*].ListenerDescriptions[*].Listener[]' --region "$region" | jq '.[] | select(.LoadBalancerPort==7999) | .InstancePort')
```

Comment:
Using the `query`/`filter` aws cli I wasn't successful in filtering out the listener with a `LoadBalancerPort` that matches `7999`. As a result Ive pulled in `jq`. I see we do have documentation that points to using `jq` so maybe this is okay? At the very least we would need to update the prerequisites to call out `jq` as a requirement.

Reply:
Just also confirmed that `jq` is available to GitHub actions.

Reply:
Added the function `check_for_prerequisites` to `install.sh` to check for prerequisites.

Reply:
Thanks @nanux for finding a solution to this - `jq` no longer needed: 
https://github.com/atlassian-labs/data-center-terraform/pull/178#r820328638